.stylelintrc

```diff
@@ -36,5 +36,8 @@
     "no-extra-semicolons": true,
     "no-missing-end-of-source-newline": true,
     "no-eol-whitespace": true
-  }
+  },
+  "ignoreFiles": [
+    "media/css/sandstone/mixins/at2x.less"
```

Comment:
Note for reviewer: I had to move the Less version of the `.at2x()` mixin to a separate file here, so that it can be ignored. The latest Stylelint throws an error when trying to parse the inline JavaScript contained inside the `at2x` function. 

I considered filing an issue over on [postcss-less](https://github.com/shellscape/postcss-less), however I also learned that inline JS [has been deprecated](http://lesscss.org/usage/#command-line-usage-options-specific-to-lessc) in Less. I decided rather than go down that rabbit hole, to make an exception here and instead concentrate on removing Less from bedrock.